lib/sacastats_web/templates/layout/root.html.heex

```diff
@@ -1,30 +1 @@
-<!DOCTYPE html>
```

Comment:
Is it okay to delete this !DOCTYPE tag? I didn't notice any issues while testing locally but just want to be sure.

Reply:
Yeah, that was an integrated HTML page, so it was just a duplicate that I removed.